packages/fxa-payments-server/.storybook/components/MockApp.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode } from 'react';
+import config from '../../src/lib/config';
+import { StripeProvider } from 'react-stripe-elements';
+import { MockLoader } from './MockLoader';
+
+type MockAppProps = {
+  children: ReactNode
+};
+
+export const MockApp = ({ children }: MockAppProps) => <>
```

Comment:
This is an attempt to simulate the overall page & app context for route components in Storybook.